src/App.tsx

```diff
@@ -15,36 +17,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  None = '',
+  Alphabetically = 'sortAlphabetically',
+  ByLength = 'sortLength',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.None);
+  const [reversed, setReversed] = useState(false);
+
+  const visibleGoods = [...goodsFromServer].sort((good1, good2) => {
+    switch (sortField) {
+      case 'sortAlphabetically':
+        return good1.localeCompare(good2);
+      case 'sortLength':
```

Comment:
Instead of using the string 'sortLength', you should use the enum value `SortType.ByLength` to ensure type safety and consistency with the rest of your code.